spec/support/views/content/_dummy.html.erb

```diff
@@ -0,0 +1 @@
+<p>Dummy content</p>
```

Comment:
I can't see this file referenced anywhere?

Reply:
Ah yeah, good spot. Remnant of a failed attempt 😅